java/src/jmri/jmrit/display/IndicatorTrackIcon.java

```diff
@@ -285,7 +282,7 @@ private void setStatus(OBlock block, int state) {
         _status = _pathUtil.getStatus(block, state);
         if ((state & (OBlock.OCCUPIED | OBlock.RUNNING)) != 0) {
             // _pathUtil.setLocoIcon must run on GUI. LocoLabel ctor causes editor to draw a graphic
-            ThreadingUtil.runOnLayoutEventually(() -> {
+            ThreadingUtil.runOnGUIEventually(() -> {
```

Comment:
I totally agree that this change is correct. But on the other hand, it doesn't change anything at all. The layout thread is the GUI thread. But we have two different methods in case we would like to split these so that the layout thread and the GUI thread would become two different threads.

https://github.com/JMRI/JMRI/blob/master/java/src/jmri/util/ThreadingUtil.java#L81

```
    static public void runOnLayoutEventually(@Nonnull ThreadAction ta) {
        runOnGUIEventually(ta);
    }
```

Reply:
Good point @danielb987